fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -160,6 +160,8 @@ def ootConvertMeshToC(originalObj, finalTransform, f3dType, isHWv1, name, folder
 		data.source += '#include "' + folderName + '.h"\n\n'
 	else:
 		data.source += '\n'
+	if bpy.context.scene.celShadingPatch:
+		data.source += '#define G_CELSHADING 0x00400000\n\n'
```

Comment:
the microcode patch should probably have this defined in `gbi.h`, or else there could be a risk of getting a bunch of redefinition errors

Reply:
`gbi.h` is provided by z64hdr in the case of Triforce%, so it's not possible to be changed in that project. Perhaps in a decomp-based toolchain people will be willing to edit their own `gbi.h` when they apply the microcode change as well. However, please keep in mind that redefining a macro to the exact same thing as it was already defined to is not an error, it's silently ignored by the preprocessor.